components/navbar/index.tsx

```diff
@@ -0,0 +1,48 @@
+import Image from "next/image";
+
+type Props = {
+  open: () => void;
+};
+
+const NavBar = ({ open }: Props) => {
+  return (
+    <>
+      <nav className="bg-background-300 px-3 py-4 tablet:px-8 desktop:px-12 flex flex-row justify-between items-center">
+        {/* logo and logo text */}
+        <div className="flex flex-row gap-2 items-center">
+          <div className="relative w-8 h-8 tablet:w-9 tablet:h-9">
+            <Image src="/cc-logo.svg" alt="court canva logo" layout="fill" objectFit="contain" />
+          </div>
+          <p className="text-p tablet:text-logoText text-typography-500">CourtCanva</p>
+        </div>
+        {/* nav bar content */}
+        <ul className="hidden tablet:block text-p text-white w-[233px] h-6 tablet:flex tablet:justify-between tablet:items-center">
```

Comment:
tablet:flex , tablet:block have same CSS properties can not apply at the same time